bin/oref0-dex-wait-until-expected.sh

```diff
@@ -10,7 +10,9 @@ if (( $(bc <<< "$TIME_SINCE >= $OLD") )); then
   echo "CGM Data $TIME_SINCE mins ago is old (>=$OLD), not waiting"
 else
   WAIT_MINS=$(bc <<< "$OLD - $TIME_SINCE")
-  if (( $(bc <<< "$WAIT_MINS >= $MAX_WAIT") )); then
+  if (( $(bc <<< "$WAIT_MINS > 6") )); then
+    echo "Clock mismatch ($WAIT_MINS > 6); not waiting"
```

Comment:
Why is this a clock mismatch? I think this also will occur when a CGM signal is missed 2 * 5> 6.

Reply:
You're thinking about it backwards.  $WAIT_MINS is the number of minutes between the rig's current time and 5.1 minutes after the last BG reading.  So if we miss one or more CGM data points, it'll be negative, and this code path won't execute.  This only shows up if the rig time is in the past (usually because it hasn't done NTP and is showing Y2K).